js/pages/Router.js

```diff
@@ -78,7 +78,7 @@ define(
         }, {});
         // anyway, we should track the moment when the user exits and check permissions once again
 				authApi.isAuthenticated.subscribe((isAuthenticated) => {
-					if (!isAuthenticated) {
+					if (!isAuthenticated && window.location.hash !== '#/feedback') {
```

Comment:
I think it is better to use this.getModel().activePage() instead of windows.location.hash
+ it also needs to be done for Home page.

Reply:
updated